mantid-developer-linux.yml

```diff
@@ -0,0 +1,68 @@
+name: mantid
+
+channels:
+  - conda-forge
+  - DLR-SC # Required for OpenCascade
+
+dependencies:
+  - python=3.8.*
+  - cmake>=3.20.*
+  - texlive-core>=20180414
+  - doxygen>=1.9.*
+  - tbb=2020.2.*
+  - tbb-devel=2020.2.*
+  - boost=1.75.* # Also pulls in boost-cpp
+  - poco=1.10.*
+  - nexus=4.4.*
+  - hdf5=1.10.*
+  - jemalloc>=5.2.1
+  - gsl=2.6 # Keep gsl a specific version to reduce changes in our fitting
+  - jsoncpp>=1.9.4,<2
+  - ninja>=1.10.2
+  - graphviz>=2.47.0
+  - librdkafka>=1.6.0
+  - eigen=3.3.*
+  - opencascade>=7.4.0
+  - zlib>=1.2.11
+  - muparser>=2.3.2
+  - qt>=5.12.9,<6
+  # Not Windows, OpenGL implementation:
+  - mesalib>=18.0.0
+  - mesa-libgl-devel-cos7-x86_64>=18.3.4
+  - mesa-libgl-cos7-x86_64>=18.3.4
+  # Needed for test suite on Linux
+  - xorg-libxcomposite>=0.4.5
+  - xorg-libxcursor>=1.2.0
+  - xorg-libxdamage>=1.1.5
+  - xorg-libxi>=1.7.10
+  - xorg-libxtst>=1.2.3
+  - xorg-libxscrnsaver>=1.2.3
+  - pciutils-libs-cos7-x86_64>=3.5.1
+  # Linux only
+  - gxx_linux-64==9.3.*
+  - libglu>=9.0.0
+  # Python Packages:
+  - setuptools=48.0.0 # Pinned purposefully due to incompatibility with later versions
+  - qtwebkit>=5.212,<6
+  - qscintilla2>=2.11.2
+  - pyqt>=5.12.3,<6
+  - qtpy>=1.9.0
+  - qtconsole>=5.0.3,<6
+  - numpy>=1.20.2
+  - scipy>=1.6.2
+  - sphinx>=3.5.4
+  - pycifrw>=4.4.1
+  - python-dateutil>=2.8.1
+  - h5py>=2.10.0,<3  # Pinned back due to api differences
+  - psutil>=5.8.0
+  - requests>=2.25.1
+  - toml>=0.10.2
+  - pyyaml>=5.4.1
+  - pip>=21.0.1
+  # Needed only for development
+  - pre-commit>=2.12.0
+  - cppcheck>=2.4.1
+  - cxxtest>=4.4
```

Comment:
We have an internal copy of ths so I don't think we need this either.

Reply:
Removed